src/components/ConfigEditor/ConfigSection.tsx

```diff
@@ -0,0 +1,63 @@
+import React, { useState } from "react";
+import { css } from "@emotion/css";
+import { useTheme2, IconButton, IconName } from "@grafana/ui";
+
+type Props = {
+  title: string;
+  description: string;
+  isCollapsible?: boolean;
+  isOpen?: boolean;
+  kind?: "section" | "sub-section";
+  className?: string;
+};
+
+export const ConfigSection: React.FC<Props> = ({
+  children,
+  title,
+  description,
+  isCollapsible = false,
+  isOpen: isInitiallyOpen = true,
+  kind = "section",
+  className,
+}) => {
+  const { colors, typography } = useTheme2();
+  const [isOpen, setIsOpen] = useState(isCollapsible ? isInitiallyOpen : true);
+  const iconName: IconName = isOpen ? "angle-up" : "angle-down";
+
+  const styles = {
+    header: css({
+      display: "flex",
+      justifyContent: "space-between",
+      alignItems: "center",
+    }),
+    subtitle: css({
+      marginBottom: typography.pxToRem(4),
+      fontWeight: typography.fontWeightRegular,
+    }),
+    descriptionText: css({
+      ...typography.bodySmall,
+      color: colors.text.secondary,
+    }),
+  };
+
+  return (
+    <div className={className}>
```

Comment:
According to design there is only line between sections, which can be added with `<hr>` I guess:

![image](https://user-images.githubusercontent.com/1436174/234573359-e84b311b-23ca-4b11-b409-82e4b673529a.png)

@catherineymgui , do you have anything to add?

Reply:
No there shouldn't be any border / shadow. But since I am deciding to introduce a new styling for collapsable sections, we may need to contribute back to the design system